document/js-api/index.bs

```diff
@@ -720,10 +720,10 @@ dictionary GlobalDescriptor {
   boolean mutable = false;
 };
 
-[LegacyNamespace=WebAssembly, Constructor(GlobalDescriptor descriptor, unrestricted double value = 0), Exposed=(Window,Worker,Worklet)]
+[LegacyNamespace=WebAssembly, Constructor(GlobalDescriptor descriptor, optional any value = undefined), Exposed=(Window,Worker,Worklet)]
```

Comment:
Web IDL currently [does not support `undefined`](https://github.com/heycam/webidl/issues/60). How about just removing `= undefined`?

Reply:
Done.